### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-req-build-c7s69ktb/requirements.txt'

### ISSUE THREAD

**Issue by skupjoe** [opsdis/monitor-promdiscovery#7]
Since commit 1bfa206, I am seeing the following error when I attempt to build the latest version of monitor-promdiscovery

```
Processing ./dist/monitor-promdiscovery-0.0.4.post4+git.1bfa2065.dirty.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-c7s69ktb/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-c7s69ktb/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(cod
e, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-ofunht7m
         cwd: /tmp/pip-req-build-c7s69ktb/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-c7s69ktb/setup.py", line 30, in <module>
        install_requires=read('requirements.txt').split(),
      File "/tmp/pip-req-build-c7s69ktb/setup.py", line 6, in read
        return open(join(dirname(__file__), fname)).read()
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-req-build-c7s69ktb/requirements.txt'
```

To reproduce, you can launch a Docker instance of Python:

```
docker run -it --rm python:3.9 /bin/sh
```

and attempt to build using these steps:

```
git clone https://github.com/opsdis/monitor-promdiscovery.git /opt/monitor-promdiscovery
cd /opt/monitor-promdiscovery
pip3 install -r requirements.txt
python3 setup.py sdist
pip3 install dist/monitor-promdiscovery*
```

**Comment by skupjoe** [comment]
Reverting to commit  5b8653c fixes the issue:

```
git clone https://github.com/opsdis/monitor-promdiscovery.git /opt/monitor-promdiscovery
cd /opt/monitor-promdiscovery
git reset --hard 5b8653cae3544e3fc372dca10f0fbf4aa48e99f5
pip3 install -r requirements.txt
python3 setup.py sdist
pip3 install dist/monitor-promdiscovery*
```

**Comment by thenodon** [comment]
Thanks for the bug report. This is fixed in commit 303a9e1 and was due to a missing MANIFEST.ini file
